pimd/pim_cmd.c

```diff
@@ -4491,69 +3198,71 @@ DEFUN (show_ip_pim_interface,
        "interface name\n"
        JSON_STR)
 {
-	int idx = 2;
-	struct vrf *vrf = pim_cmd_lookup_vrf(vty, argv, argc, &idx);
-	bool uj = use_json(argc, argv);
-	bool mlag = false;
+	struct vrf *vrf_struct;
+	bool uj = !!json;
```

Comment:
What is this symbol "!!"

Reply:
As we are assigning const char* to bool, safe to use !!, (followed from Donald's changes)